src/cond.d

```diff
@@ -89,18 +89,63 @@ extern (C++) class DVCondition : Condition
  */
 extern (C++) final class DebugCondition : DVCondition
 {
+    /**
+     * Set the global debug level
+     *
+     * Only called from the driver
+     *
+     * Params:
+     *   level = Integer literal to set the global version to
+     */
     static void setGlobalLevel(uint level)
     {
         global.params.debuglevel = level;
     }
 
+
+    /**
+     * Add an user-supplied identifier to the list of global debug identifiers
+     *
+     * Can be called from either the driver or a `debug = Ident;` statement.
+     * Unlike version identifier, there isn't any reserved debug identifier
+     * so no validation takes place.
+     *
+     * Params:
+     *   ident = identifier to add
+     */
+    deprecated("Kept for C++ compat - Use the string overload instead")
```

Comment:
I kept it (and the one on `VersionCondition`) because a quick search on LDC yielded some results:
- https://github.com/ldc-developers/ldc/blob/d7d94c5a71a97f60ecb9cc0ece6a36342191f7da/driver/main.cpp#L501-L504
- https://github.com/D-Programming-GDC/GDC/blob/cb1e58b0da3ef9003bb89331af3c31bd00cddfb0/gcc/d/d-lang.cc#L372-L387